app/jobs/subscription_placement_job.rb

```diff
@@ -62,6 +62,10 @@ def cap_quantity_and_store_changes(order)
     unavailable_stock_lines_for(order).each do |line_item|
       changes[line_item.id] = changes[line_item.id] || line_item.quantity
       line_item.update(quantity: 0)
+
+      Spree::OrderInventory.new(order).verify(line_item, order.shipment)
+      order.line_items.reload
+      order.recreate_all_fees!
```

Comment:
This might be a bit expensive, especially if there are multiple out of stock items. I think we can get the same outcome with this (but with a lot less transactions):
```ruby
    unavailable_stock_lines_for(order).each do |line_item|
      changes[line_item.id] = changes[line_item.id] || line_item.quantity
      line_item.update(quantity: 0)

      Spree::OrderInventory.new(order).verify(line_item, order.shipment)
    end
    if changes.present?
      order.line_items.reload
      order.update_order_fees!
    end
```

Reply:
👍 done. Good catch.